shop-local/src/components/Store-info.js

```diff
@@ -1,28 +1,36 @@
 import "../App.css"
+import React, { useState, useEffect } from 'react';
 
 
-// const StoreInfo = ({ name, address, description, openTime, closeTime, phoneNumber, instagram, facebook }) => {
-//     if (!name) return <div />;
-//     return (
-//         <div className='store-container'>
-//             <h2 className='store-name'>{name}</h2>
-//             <h3 className='store-info'>{address}</h3>
-//             <p className='store-info'>{description}</p>
-//             <p className='store-info'>{openTime}-{closeTime}</p>
-//             <p className='store-info'>Phone Number:{phoneNumber}</p>
-//             <a className='store-info' href={instagram}>Instagram</a>
-//             <a className='store-info' href={facebook}>Facebook</a>
-//         </div>
-//     )
-// }
+function Store({ match }) {
```

Comment:
Might be helpful to add some comments about what the functions do for future reference